src/Microsoft.PowerShell.ConsoleHost/host/msh/Telemetry.cs

```diff
@@ -21,6 +21,7 @@ internal static class ApplicationInsightsTelemetry
         // The name of the file by when present in $PSHOME will enable telemetry.
         // If this file is not present, no telemetry will be sent.
         private const string TelemetrySemaphoreFilename = "DELETE_ME_TO_DISABLE_CONSOLEHOST_TELEMETRY";
+        private const string TelemetryOptout = "POWERSHELL_TELEMETRY_OPTOUT";
```

Comment:
Need rename the file in root repo folder.

Reply:
This is used for the environment variable not the file

Reply:
Perhaps it should be called TelemetryOptoutEnvVar

Reply:
See a54f7dc where I have renamed the constant as you proposed.